build 8a2dcec09 fails to compile

openSUSE Tumbleweed 
VERSION_ID="20220916"

/usr/bin/msgfmt: found 4 fatal errors
573 translated messages, 1141 fuzzy translations, 1341 untranslated
/messages.
gmake[2]: *** [po/CMakeFiles/locale_ro.dir/build.make:73:
/share/locale/ro/LC_MESSAGES/darktable.mo] Error 1
gmake[2]: *** Deleting file 'share/locale/ro/LC_MESSAGES/darktable.mo'
gmake[1]: *** [CMakeFiles/Makefile2:2041: po/CMakeFiles/locale_ro.dir/all]
Error 2
gmake[1]: *** Waiting for unfinished jobs....
/data/dt-build/darktable/po/pt_BR.po:6: warning: header field
/'Project-Id-Version' still has the initial default value
4613 translated messages.
4614 translated messages, 1 fuzzy translation.
4615 translated messages.
4394 translated messages.
3718 translated messages.
4607 translated messages.
3871 translated messages.
4615 translated messages.
4615 translated messages.
gmake: *** [Makefile:156: all] Error 2

failing on four similar machines

--- a/po/pt_BR.po
+++ b/po/pt_BR.po
@@ -4,7 +4,7 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PACKAGE VERSION\n"
+"Project-Id-Version: darktable\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2022-09-16 14:12-0300\n"
 "PO-Revision-Date: 2022-09-16 14:13-0300\n"

--- a/po/ro.po
+++ b/po/ro.po
@@ -1579,7 +1579,7 @@ msgid ""
 "<span style=\"italic\">%s</span>\n"
 "\n"
 "do you want to proceed or quit now to do a backup\n"
-msgstr "continuati sau anulati si faceti backup"
+msgstr ""
 
 #: ../src/common/database.c:1782
 msgid "darktable - schema migration"
@@ -1601,10 +1601,7 @@ msgid ""
 "it seems that the database is corrupt.\n"
 "do you want to close darktable now to manually restore\n"
 "the database from a backup or start with a new one?"
-msgstr "se pare ca baza de date este corupta.\n" 
-"doriti sa inchideti darktable si sa efectuati o restaurare\n" 
-"manuala a bazei de date, din backup, sau sa initiati o baza de date noua?\n"
-
+msgstr ""
 
 #: ../src/common/database.c:1968 ../src/common/database.c:2046
 msgid "darktable - error opening database"

